packages/expo-modules-core/android/src/main/cpp/ExpoModulesHostObject.cpp

```diff
@@ -33,6 +33,15 @@ void ExpoModulesHostObject::set(jsi::Runtime &runtime, const jsi::PropNameID &na
 }
 
 std::vector<jsi::PropNameID> ExpoModulesHostObject::getPropertyNames(jsi::Runtime &rt) {
-  return {}; // TODO(@lukmccall): get list of all modules
+  auto names = installer->getModulesName();
+  size_t size = names->size();
+  std::vector<jsi::PropNameID> result;
+  result.reserve(size);
```

Comment:
Isn't it the same as
```suggestion
  std::vector<jsi::PropNameID> result(size);
```
? 🤔